layouts/shortcodes/contributors.html

```diff
@@ -0,0 +1,32 @@
+{{ $urlPre := "https://api.github.com/repos" }}
+{{ $username := "Apache" }}
+{{ $repo := "camel-website"}}
+{{ $token := ""}}
+<!-- Enter token here -->
+{{ $scope := "all"}}
+{{ if eq $token "" }}
+<p>Unable to get contributors details[set token]</p>
+{{else}}
+{{ $collaborators := getJSON $urlPre "/" $username "/" $repo "/collaborators?access_token=" $token  }}
```

Comment:
Needs to be cleaned up considerably, based on the last discussion we had token is not required; this also has unused variables. Could be made considerably simpler.

Reply:
I have been able to pull the Contributors data and have it display on the page but I am concerned that the names retrieved from this data do not correspond with the names listed in the Contributors section on the team.md page. Is there somewhere else to pull this data from or can I go on with what I have? Thank you.

Reply:
There are two sources of data: from apache.org and GitHub. From apache.org we can get the committers and project management committee members and from GitHub we can get contributors. On GitHub some of the contributors are also committers and project management committee members.